### PR TITLE
Add http api docker image

### PR DESCRIPTION
**Pull request by rudigiesler**

### CODE REVIEW

**Comment by rudigiesler** [comment]
Ready for review

File: docker-vms/http_api/api_tokens.yaml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Example token. Replace for production
+-
+  account: "9e60a884-4b59-44db-a8a4-52c4bb02a874"
+  conversation: "2742b45a-ef7c-43f9-b782-5af8c4cc7e7a"
+  tokens: ["563735b0ea62"]
```
Comment (hodgestar) [inline docker-vms/http_api/api_tokens.yaml:5]: Hmm. Shouldn't the account and authentication tokens also be set via environment variables?

Comment (hodgestar) [inline docker-vms/http_api/api_tokens.yaml:5]: Ah, I missed the comment at the top.

Comment (hodgestar) [inline docker-vms/http_api/api_tokens.yaml:5]: How do we plan to set these in production?

**Comment by hodgestar** [comment]
Left one slightly confused comment, otherwise looks good. :)

File: docker-vms/http_api/api_tokens.yaml (hunk 1 above)
Comment (rudigiesler) [inline docker-vms/http_api/api_tokens.yaml:5]: I was planning on having a tokens file (say `vumi_api_tokens.yaml`). Then the docker command will be `docker run --rm -v /...../vumi_api_tokens.yaml:/app/api_tokens.yaml praekelt/mama-ng-http-api`. I've tested this locally and it seems to replace the file.

Comment (hodgestar) [inline docker-vms/http_api/api_tokens.yaml:5]: Woot.

**Comment by hodgestar** [comment]
:+1:

**Pull request merged**